lib/membrane_matroska/muxer.ex

```diff
@@ -52,9 +51,11 @@ defmodule Membrane.Matroska.Muxer do
 
   defmodule State do
     @moduledoc false
+    use Bunch.Access
+
     defstruct tracks: %{},
               segment_position: 0,
-              expected_tracks: 0,
+              expected_tracks: %{},
```

Comment:
How about `active_tracks_cnt` and `expected_tracks`?

Reply:
Yeah, I would definitely go for `active_tracks_cnt`. However, since `expected_tracks` is a mapping of a form `track_id => track_ordering_number`, I think it could be worth to somehow reflect that fact in the name...how about `expected_tracks_order_map`?